tests/app/celery/test_contact_information_tasks.py

```diff
@@ -91,19 +97,24 @@ def test_should_not_retry_on_non_retryable_exception(client, mocker, notificatio
         'app.celery.contact_information_tasks.update_notification_status_by_id'
     )
 
-    with pytest.raises(Exception) as exc_info:
+    with pytest.raises(NotificationPermanentFailureException) as exc_info:
         lookup_contact_info(notification.id)
 
-    assert exc_info.type is NotificationPermanentFailureException
     mocked_va_profile_client.get_email.assert_called_with(EXAMPLE_VA_PROFILE_ID)
 
     mocked_update_notification_status_by_id.assert_called_with(
         notification.id, NOTIFICATION_PERMANENT_FAILURE, status_reason=exception.failure_reason
     )
+    mocked_check_and_queue_callback_task.assert_called_once_with(notification)
 
 
 @pytest.mark.parametrize('exception_type', (Timeout, VAProfileRetryableException))
-def test_should_retry_on_retryable_exception(client, mocker, notification, exception_type):
+def test_should_retry_on_retryable_exception(client, mocker, sample_template, sample_notification, exception_type):
```

Comment:
"" test naming

Reply:
We are changing our approach to testing.